### ISSUE TITLE
How can I set the data set of each training to be the same in this project? 

### ISSUE THREAD

**Issue by liu-zhi97** [facebookresearch/detectron2#1852]
I want to set up the same training set to fully replicate the last session.The images of the same iteration for the first training and the second training are the same. Or is there any other way to fully replicate it?If anyone knows, please let me know.

**Comment by ppwwyyxx** [comment]
Setting `cfg.SEED` to a fixed positive value and `cfg.DATALOADER.NUM_WORKERS=0` should fix the order of images.
